packages/dashboard/src/components/tests/appbar.test.tsx

```diff
@@ -1,21 +1,29 @@
+import { ThemeProvider } from '@material-ui/core';
 import { render } from '@testing-library/react';
 import { act } from '@testing-library/react-hooks';
 import userEvent from '@testing-library/user-event';
 import React from 'react';
-import { AppController, AppControllerContext } from '../app-contexts';
+import { MemoryRouter } from 'react-router';
```

Comment:
Interesting, I didn't know about it https://testing-library.com/docs/example-react-router/#reducing-boilerplate

Reply:
oh nice, I didn't know about it as well. I just sort of hack it by wrapping the `render` function.